client/lib/sites-list/test/index.js

```diff
@@ -85,6 +85,16 @@ describe( 'SitesList', () => {
 				assert.deepEqual( updatedValue, site[ prop ] );
 			} );
 		} );
+
+		it( 'should not trigger change when sites attributes didn\'t change', () => {
+			const changeCallback = sinon.spy();
+
+			sitesList.once( 'change', changeCallback );
+
+			sitesList.update( cloneDeep( data.original ) );
+
+			assert.isFalse( changeCallback.called );
+		} );
```

Comment:
Hmm, not sure I understand you 100%. :) I didn't touch the test fixtures (=> [test data](https://github.com/Automattic/wp-calypso/blob/228b91315aefb2bb74c6fcde434da35babe8abd7/client/lib/sites-list/test/fixtures/data.js)), only added the above test. It was failing before I removed the `is_previewable` etc from site schema though.

> Could we also add one of the computed props to the test data is_previewable, is_customizable

Yes, will do.